src/main/java/org/simplify4u/plugins/ArtifactResolver.java

```diff
@@ -358,11 +358,11 @@ private Artifact resolveSignature(Artifact artifact, SignatureRequirement requir
         final ArtifactResolutionResult resolution = repositorySystem.resolve(request);
         if (!resolution.isSuccess()) {
             if (resolution.hasMissingArtifacts()) {
-                log.warn("Missing artifacts for " + artifact.getId() + ": " + resolution.getMissingArtifacts());
+                LOG.warn("Missing artifacts for {}: {}", artifact.getId(), resolution.getMissingArtifacts());
             }
             resolution.getExceptions().forEach(e -> {
-                log.warn("Failed to resolve transitive dependencies for " + artifact.getId() + ": " + e.getMessage());
-                log.debug(e);
+                LOG.warn("Failed to resolve transitive dependencies for {}: {}", artifact.getId(), e.getMessage());
+                LOG.debug("Exceptions in artifact resolution.", e);
```

Comment:
Remove duplicate logging in debug level

Reply:
There is no method available that only accepts a throwable to log just the exception. That's why I added the line. If we remove the line in total, we do not have any exception info at all. Any ideas?
I mean, we can always remove the debug-logging in total. In that case we have the error _message_ but not the complete stacktrace.

Reply:
I thought, that we can remove log with debug and exception stacktrace at all.
I think that stacktrace in this point is not really helpful.